tests/vm.test.ts

```diff
@@ -626,17 +661,18 @@ test('attachments', async () => {
 
 test('runEvery', async () => {
     const indexJs = `
-        function runEveryMinute(meta) {
+        function runEveryMinute (meta) {
```

Comment:
Why did the space appear here? 🤔

Reply:
To be consistent with other function declarations and prettier (which doesn't format inside strings).
Basically I was CMD+F'ing and couldn't find every place this `runEveryMinute` was created.